models/apartment.rb

```diff
@@ -0,0 +1,26 @@
+class Apartment < ActiveRecord::Base #inherits from Active Record
+  has_many :tenants
+  attr_accessor :renters
+  attr_reader :address, :monthly_rent, :sqft, :numb_beds, :num_baths
```

Comment:
remove the `attr` helper methods here and let Active Record define getters and setters